user-guide/Basic_Functionality/Visio/turning_shapes_into_controls/Adding_options_to_a_parameter_control.md

```diff
@@ -125,13 +125,14 @@ Example:
 IncludeWrite|Lite|CustomColors:text=#0000FF,text.disabled=#8888FF,text.titlesuffix=#FF0000, bg.hover=#FFFF00,bg.hovereditor=#66FF66
 ```
 
-| Color definition | Description                                                                                     |
-| ---------------- | ----------------------------------------------------------------------------------------------- |
-| text             | The color of the default text.                                                                  |
-| text.disabled    | The color of the text when the value is "not initialized".                                      |
-| text.titlesuffix | The color of the table index suffix in the title.                                               |
-| bg.hover         | The color of the background when hovering the mouse cursor over the control.                    |
-| bg.hovereditor   | The color of the background when hovering the mouse cursor over the editor part of the control. |
+| Color definition | Description                                                                                                          |
+| ---------------- | -------------------------------------------------------------------------------------------------------------------- |
+| text             | The color of the default text.                                                                                       |
+| text.disabled    | The color of the text when the value is "not initialized".                                                           |
+| text.titlesuffix | The color of the table index suffix in the title.                                                                    |
+| bg.hover         | The color of the background when hovering the mouse cursor over the control.                                         |
+| bg.hovereditor   | The color of the background when hovering the mouse cursor over the editor part of the control.                      |
+| bg.pressededitor | As of 10.3.9, the color of the background when pressing the left mouse button within the editor part of the control. |
```

Comment:
Can you also specify the main release version? In addition, when possible we try to add the RN number itself in comment, so that it's easier to find where a particular RN has been added in the documentation.

For example: 
```suggestion
| bg.pressededitor | As of DataMiner 10.3.9/10.4.0<!-- RN 36779 -->, the color of the background when the left mouse button is pressed within the editor part of the control. |
```